systemFlake.nix

```diff
@@ -122,7 +149,9 @@ let
           })
         ] ++ host.modules;
         specialArgs = host.specialArgs;
-      };
+      } // (optionalAttrs (host.output == "nixosConfigurations") {
```

Comment:
I guess such comparison is fine, but I'd love to hear if anyone else has better ideas to detect nixos system

Reply:
I think it would be better if we this was done with the custom builder argument.

Reply:
Moving this to the builder seems like a good idea. I'd argue for it to be the default, though.

Reply:
I feel like a custom builder would make it more complex than it needs to be for the end-user, maybe we should keep current implementation?